_data-prepper/pipelines/configuration/processors/trace-peer-forwarder.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Trace peer forwarder
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Trace peer forwarder
+
+The `Trace peer forwarder` processor is used to reduce the number of events that are forwarded in a [Trace analytics]({{site.url}}{{site.baseurl}}/data-prepper/common-use-cases/trace-analytics/) pipeline by half when using [Peer forwarder]({{site.url}}{{site.baseurl}}/data-prepper/managing-data-prepper/peer-forwarder/). It groups the events based on `trace_id` similar to the `service_map_stateful` and `otel_trace_raw ` processors. 
```

Comment:
1. Event is duplicated when it goes between different pipelines. 
2. Helps because it allows us to duplicate once in the entry pipeline (raw-pipeline) instead of the other (service-map-pipeline). Prevents additional, unnecessary HTTP requests.
3. Performance benefit.